code/controllers/subsystem/sounds.dm

```diff
@@ -59,7 +62,35 @@ SUBSYSTEM_DEF(sounds)
 		".aiff",
 	)
 
-	all_sounds = pathwalk("sound/", valid_file_extensions)
+	all_sounds += pathwalk("sound/ambience", valid_file_extensions)
+	all_sounds += pathwalk("sound/music", valid_file_extensions)
+	all_sounds += pathwalk("sound/announcer", valid_file_extensions)
+
+	/datum/controller/subsystem/sounds/proc/find_all_sfx_except_long()
```

Comment:
how do we know these aren't long

Reply:
This tbh, but I wouldn't be surprised if there wasn't an easy way to check it.